### ISSUE TITLE
Final Item Ingredients

### ISSUE THREAD

**Issue by Zak223** [Zak223/Greggic-Skies#19]
- [ ] thaumcraft:creative_flux_sponge - Flux Sponge
- [ ] storagedrawers:upgrade_creative:0 - Creative storage upgrade
- [ ] appliedenergistics2:creative_storage_cell - Creative ME Storage Cell
- [ ] thermalexpansion:capacitor:32000 - Flux Capacitor (Creative)
- [ ] mysticalagradditions:stuff:69 - Creative Essence
- [ ] draconicevolution:draconium_capacitor:2 - Creative Flux Capacitor
- [ ] draconicevolution:creative_rf_source - Creative RF Source
- [ ] appliedenergistics2:creative_energy_cell - Creative Energy Cell
- [ ] gregtech:machine:2102 - Infinite Pressure Pump
- [ ] galacticraftcore:infinite_oxygen - Infinite Oxygen Supply
- [ ] gregtech:machine:999 - MAX Machine Hull
- [ ] gcys:meta_item_1:34 - Supra-Chronal Maxium Circuit (MAX Circuit)

**Comment by MASONHAA** [comment]
i dont think you should have the creative storage upgrade and creative me storage cell as part of this recipe 
it would just be broken because they can give you infintie items

**Comment by Zak223** [comment]
> i dont think you should have the creative storage upgrade and creative me storage cell as part of this recipe 
> it would just be broken because they can give you infintie items

That's a good point, I knew I was going to rework the recipe so I'm going to add new items in place of those.

Might add some contenttweaker items also
